kinoml_env.yml

```diff
@@ -0,0 +1,197 @@
+name: kinoml
```

Comment:
Do we want to pull in this frozen environment? I think we want our environment YAML files to be minimal?